src/xunit.performance.api/XunitPerformanceHarness.cs

```diff
@@ -52,19 +54,90 @@ private void ProcessResults()
         {
             CSVMetricReader reader = new CSVMetricReader(Configuration.FileLogPath);
 
-            foreach(string testCaseName in reader.TestCases)
+            foreach (string testCaseName in reader.TestCases)
             {
                 Console.WriteLine($"\nTest Case: {testCaseName}");
                 List<Dictionary<string, double>> iterations = reader.GetValues(testCaseName);
-                for(int i=0; i<iterations.Count; i++)
+
+                for (int i = 0; i < iterations.Count; i++)
                 {
                     Dictionary<string, double> iter = iterations[i];
-                    foreach(KeyValuePair<string, double> result in iter)
+                    foreach (KeyValuePair<string, double> result in iter)
                     {
                         Console.WriteLine($"Iter={i}; Metric={result.Key}; Value={result.Value}");
                     }
                 }
             }
+
+            WriteStatisticsToFile(reader);
+        }
+
+        /// <summary>
+        /// This method writes the performance results to csv files.
+        /// </summary>
+        /// <param name="reader"></param>
+        private void WriteStatisticsToFile(CSVMetricReader reader)
+        {
+            //Console.WriteLine();
+            //Console.Write("[DEBUG] Press <Enter> to continue...");
+            //Console.ReadLine();
+
+            /*
+             * Generate CSV data (Probably not the most efficient way, e.g. How big can this become).
+             *  Test Name,Metric,Iterations,Average,Sample standard deviation,Minimum,Maximum
+             */
+            //var rawFilePath = $"{Configuration.RunId}-raw.csv";
+            //var rawTable = new DataTable();
+
+            var summaryFilePath = $"{Configuration.RunId}-Summary.csv";
+            var summaryTable = new DataTable();
+            var col0 = summaryTable.AddColumn("Test Name");
```

Comment:
Can you rename the column variables to be more descriptive?  It will make sure that down below when they are used that they match the data that's actually being put into them.  I'm sure it works now, but this is just a maintainability issue for later.